internal/c/libqb.cpp

```diff
@@ -37126,9 +37126,9 @@ qbs *func__droppedfile(int32 fileIndex, int32 passed) {
 void sub__resize(int32 on_off, int32 stretch_smooth) {
 
     if (on_off == 1)
-        resize_snapback = 0;
+        resize_snapback = 0; //ON

```

Comment:
What does `OFF` and `ON` mean in this context?

Also, FWIW, we should look to replace a lot of these with `enum`s, then we can just name the values it can be.

Reply:
It's counterintuitive in this case.  When we set $RESIZE:ON in a program (or _RESIZE ON), snapback is 0.  $RESIZE:OFF sets it to 1.

snapback is the process where our window snaps back to its original size, if we try to scale it with $RESIZE ON and then _RESIZE OFF.  We don't refuse to scale, which would make sense to me...  We just scale then reset/snapback.